packages/editor/src/store/utils/notice-builder.js

```diff
@@ -65,7 +66,7 @@ export function getNotificationArgumentsForSaveSuccess( data ) {
 	const actions = [];
 	if ( shouldShowLink ) {
 		actions.push( {
-			label: postType.labels.view_item,
+			label: isDraft ? 'View Preview' : postType.labels.view_item,
```

Comment:
Thanks for working on this!

The 'View Preview' text will need to be translated, so should use the `__` function (e.g. `__( 'View preview' )`).

My understanding is that it should be a lower-case 'p' for preview, as I think this counts as a feature label. Most of the copy in Gutenberg uses sentence case anyway, so that's usually the best way to go.

Here's a link to the copy guidelines for capitalization where it mentions feature labels and feature names - https://developer.wordpress.org/block-editor/contributors/documentation/copy-guide/#five-pay-attention-to-capitalization.

Reply:
Thank you for the quick feedback and for sharing the guidelines! I have added the function `__` to 'View Preview'. I am now looking into how to test the translation locally. 

In regards to the casing, the existing links in the snackbar (like View Post/Page) are capitalized whereas the `noticeMessage` strings (like Draft saved/Page published) are in sentence case. Would it make sense to stay consistent with the link casing coming from core or is it preferred to match the sentence casing in the `noticeMessage`? The third screenshot I shared above shows how the casing currently looks for an existing snackbar notice with a link: 

<img width="254" alt="Screen Shot 2022-03-15 at 2 34 01 PM" src="https://user-images.githubusercontent.com/35543432/158476791-88f01a7b-5632-48b4-9bb0-83775be9f38b.png">